packages/starlight-typedoc/tests/unit/typedoc.test.ts

```diff
@@ -67,6 +72,21 @@ test('should generate the doc in `src/content/docs/api` by default', async () =>
   expect(mkdirSyncSpy.mock.calls[0]?.[0].toString()).toMatch(/src[/\\]content[/\\]docs[/\\]api$/)
 })
 
+test('should generate the doc in `/content/docs/api` of the srcDir via the AstroConfig', async () => {
+  await generateTestTypeDoc(
+    {
+      ...starlightTypeDocOptions,
+      entryPoints: ['../../fixtures/basics/src/functions.ts'],
+    },
+    { srcDir: new URL('docs/src', import.meta.url) } as AstroConfig,
+  )
+
+  const mkdirSyncSpy = vi.mocked(fs.mkdirSync)
+
+  expect(mkdirSyncSpy).toHaveBeenCalled()
+  expect(mkdirSyncSpy.mock.calls[0]?.[0].toString()).toMatch(/docs[/\\]src[/\\]content[/\\]docs[/\\]api$/)
```

Comment:
```suggestion
  expect(mkdirSyncSpy.mock.calls[0]?.[0].toString()).toMatch(/www[/\\]src[/\\]content[/\\]docs[/\\]api$/)
```

To match the previous change.